content/rs/references/crdb-cli-reference.md

```diff
@@ -0,0 +1,284 @@
+
+### Creating an Active-Active Database
+
+The `crdb create` command lets you create a customized Active-Active database and create instances of the database on specified participating clusters. Before you create an Active-Active database you must have:
+
+- At least two participating clusters
+- [Network connectivity]({{< relref "/rs/administering/designing-production/networking/port-configurations.md" >}}) between the participating clusters
+
+When you create an Active-Active database, you must specify:
+
+|Flag and argument| Argument type| Description|
+|---|---|---|
+|`name <CRDB_name>`| string| Name of the Active-Active database|
+|`memory-size <size>`| string| Maximum memory in bytes| kilobytes (kb)| or gigabytes (gb)|
+|`port <port_number>`| integer| TCP port for the Active-Active database on all participating clusters|
+|`instance fqdn=<cluster_fqdn>,username=<username>,password=<password>`| strings| The connection information for the participating clusters. You must specify this flag for each participating cluster.|
+
+The syntax for the command is:
+
+```sh
+crdb-cli crdb create --name <name> /
+--memory-size <maximum_memory> --port <port_number> /
+--instance fqdn=<cluster_fqdn>,username=<username>,password=<password> /
+--instance fqdn=<cluster_fqdn>,username=<username>,password=<password> /
+[--no-wait] /
+[--db-config-json <file_content>] /
+[--db-config-json-file <filename>] /
+[--compression (0-6)] /
+[--causal-consistency true] /
+[--password <password>] /
+[--replication true] /
+[--encryption true] /
+[--sharding false] /
+[-- shards-count <number_of_shards>] /
+[--shard-key-regex <regex_rule>] /
+[--oss-sharding BOOLEAN] /
+[--oss-cluster true]
+[--bigstore true]
+[--bigstore-ram-size <maximum_memory>]
+[--with-module <module_name>]
+```
+
+After you run the command, the response shows the GUID of the task and the GUID of the Active-Active database that was created:
+
+```sh
+Task 2b3d62c6-395b-4679-87b1-eef006ad4a38 created
+  ---> Status changed: queued -> started
+  ---> CRDB GUID Assigned: crdb:552bbccb-99f3-4142-bd17-93d245f0bc79
+  ---> Status changed: started -> finished
+```
+
+If you only specify the required fields, the properties of the database are::
+
+- Compression level 3
+- Causal consistency is off
+- No password
+- No replication
+- No encryption
+- 1 shard
+- Shard key regex: `{u'regex': u'.*\\{(?<tag>.*)\\}.*'}, {u'regex': u'(?<tag>.*)'}`
+
+The `crdb create` command supports several additional options:
+
+|Flag and argument| Argument type| Description|
+|---|---|---|
+|`no-wait`| no arguments| Prevents CRDB-CLI from running another command before this command finishes|
+|`compression (0-6)`| integer| The level of data compression: 0=Compression disabled| 1=Low compression and resource load| 6=High compression and resource load (Default: 3)|
+|`causal-consistency true`| boolean| Applies updates to all instances in the order they were received|
```

Comment:
We should link here:
https://docs.redislabs.com/latest/rs/administering/database-operations/causal-consistency-crdb/